tempus_fugit_minecraft/model.py

```diff
@@ -120,6 +121,7 @@ def _initialize(self, immediate=False) -> None:
 
         clouds = self.generate_clouds_positions(n, num_of_clouds=150)
         self.place_cloud_blocks(clouds)
+        self.generate_trees(num_trees=50)
```

Comment:
Maybe instead of setting a set number of trees per world, it could have a degree of randomness? You can either do that in this pr or add it as a an issue for the future.

Reply:
Nice thought. Before setting a default value, I did it randomly. BUT then change it.
I prefer making it in a new PR since there are many minor things I want to do them together including this one, if you don't mind.